lib/git_worktree.rb

```diff
@@ -25,6 +25,19 @@ def initialize(options = {})
     @remote_name = 'origin'
     @base_name   = File.basename(@path)
 
+    # For the `ssh` feature to be available, it requires at gem installation
+    # that libssh2 is available
```

Comment:
I didn't suffer through Sir Ronald Syme's *The Roman Revolution* in graduate school only to be further tortured like this!

"The libssh2 library must already be installed at gem installation time for the 'ssh' feature to be available."

Reply:
I saved you further agony by just doing a copy pasta this time...

**I HOPE YOU ARE SATISFIED, CAESAR!**